### ISSUE TITLE
OJS 3.3 - Importing XML issues returning Uncaught Error: Call to undefined function starts_with()

### ISSUE THREAD

**Issue by israelcefrin** [pkp/pkp-lib#6590]
**Describe the bug**
Importing XML issues is crashing and doesn't import issues from this format.

**To Reproduce**
1. Go to 'OJS folder.'
2. Run PHP  `tools/importExport.php NativeImportExportPlugin import issue-file.xml [JOURNAL_ACRONYM] [USERNAME]`
3.  See the error

**What application are you using?**
 OJS version 3.3

**Additional information**

Log error:

```
PHP Fatal error:  Uncaught Error: Call to undefined function starts_with() in testing-3_3/plugins/importexport/native/          NativeImportExportPlugin.inc.php:489
Stack trace:
#0 testing-3_3/plugins/importexport/native/NativeImportExportPlugin.inc.php(329): NativeImportExportPlugin->parseOpts(Array,    Array)
#1 testing-3_3/tools/importExport.php(63): NativeImportExportPlugin->executeCLI('tools/importExp...', Array)
#2 testing-3_3/tools/importExport.php(69): importExport->execute()
#3 {main}
  thrown in testing-3_3/plugins/importexport/native/NativeImportExportPlugin.inc.php on line 489
```

**Comment by asmecher** [comment]
Assigning to you, @defstat!

**Comment by NateWr** [comment]
Was that maybe intended to be `str_starts_with`? It looks like that is only available in PHP 8: https://www.php.net/manual/en/function.str-starts-with.php

**Comment by asmecher** [comment]
No, I think it's one of the Laravel helpers: https://laravel.com/docs/5.2/helpers

**Comment by defstat** [comment]
@asmecher 

**PRs**
[OJS]: pkp/ojs/pull/3017
[OMP]: pkp/omp/pull/918

Those changes are also added to #6490 (see https://github.com/pkp/pkp-lib/pull/6491/files#diff-5a198b5180c92443e519000b8662280e0be2114da2641c1c8149055ab8cffc8cR117). The `parseOpts` function is added to the common library.

Also there are more fixes regarding the CLI tool in #6490

**Comment by asmecher** [comment]
Merged -- thanks, @defstat!
